src/compiler/checker.ts

```diff
@@ -210,6 +210,10 @@ namespace ts {
             getSuggestionForNonexistentProperty,
             getSuggestionForNonexistentSymbol,
             getBaseConstraintOfType,
+            getJsxNamespaceSymbol: node => {

```

Comment:
After reading through, I think I see why I was confused. JSX is *usually* a global, but in theory it could be locally imported. In that case, maybe `resolveJsxNamespaceAtLocation`